packer/bsd/netbsd_vanilla.pkr.hcl

```diff
@@ -0,0 +1,113 @@
+variable "bucket" { type = string }
+variable "gcp_project" { type = string }
+variable "image_date" { type = string }
+variable "image_name" { type = string }
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+source "qemu" "qemu-gce-builder" {
+
+  boot_command            = [
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<down><enter><wait6s>",
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<enter><wait6s>",
+      "<down><enter><wait6s>",
+      "<enter><wait6s>",
+      "<down><enter><wait6s>",
+      "b<wait1s><enter><wait6s>x<enter><wait6s>",
+      "<down><enter><wait6s>",
+      "<enter><wait240s>",
+      "<enter><wait6s>",
+      "g<wait1s><enter><wait6s>",
+      "o<wait1s><enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s><enter><wait6s>b<enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s>packer<enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s>",
+      "d<wait1s><enter><wait6s><enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s>",
+      "packer<enter><wait6s>",
+      "x<wait1s><enter><wait6s>",
+      "<enter><wait6s>",
+      "d<wait1s><enter><wait120s>",
+      "r<wait1s>o<wait1s>o<wait1s>t<wait1s>",
+      "<enter><wait6s>",
+      "p<wait1s>a<wait1s>c<wait1s>k<wait1s>e<wait1s>r<wait1s>",
+      "<enter><wait6s>",
+      "sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config<enter><wait6s>",
+      "echo '/sbin/dhcpcd -4' > /etc/rc.local<enter><wait1s>",
```

Comment:
This probably isn't the right method. Perhaps @macdice knows?

Reply:
That is especially for forcing ipv4, otherwise NetBSD's ipv6 is not working correctly with QEMU. Installation of one packages takes around 30 minutes. Related issue: https://gitlab.com/qemu-project/qemu/-/issues/985